src/app/containers/Article/index.test.jsx

```diff
@@ -43,16 +48,30 @@ describe('ArticleContainer', () => {
     });
 
     describe('non-200 status code', () => {
-      shouldShallowMatchSnapshot(
+      shouldMatchSnapshot(
         'should render correctly',
         <ArticleContainer data={badData} bbcOrigin={bbcOrigin} />,
       );
     });
 
     describe('no data or bbcOrigin', () => {
-      shouldShallowMatchSnapshot(
+      shouldMatchSnapshot(
+        'should render correctly',
+        <ArticleContainer data={newsProps} error="error" />,
+      );
+    });
+
+    describe('loading state', () => {
```

Comment:
Still dubious of these tests, as really they're testing the PageHanders again

Would be interested in others opinions, @pjlee11 @sareh

Reply:
Going to look at removing the loading state test from here and mocking the HOC's